### ISSUE TITLE
Completion & linting error

### ISSUE THREAD

**Issue by eafpres** [spyder-ide/spyder#14177]
### Problem Description

![image](https://user-images.githubusercontent.com/30508983/98569896-0fb2f500-2270-11eb-9ba9-60ae74eda131.png)

### What steps reproduce the problem?

1.  Built pip only env using Spyder 4.1.5
2.  Message reported from build:
```
python-language-server 0.36.1 has requirement jedi<0.18.0,>=0.17.2, but you'll have jedi 0.17.1 which is incompatible.
```
however, using jedi > 0.17.1 produces a conflict error w/Spyder 4.1.5
3. Run Spyder (in this case, execute Spyder3 from PowerShell

### What is the expected output? What do you see instead?

Absence of the error, linting & completion enabled  .

### Paste Traceback/Error Below (if applicable)

```python-traceback

![image](https://user-images.githubusercontent.com/30508983/98570394-a2539400-2270-11eb-9a9d-1b6e62109403.png)


```

## Versions
* Spyder version: 4.1.5 None
* Python version: 3.7.4 64-bit
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10
```

```
## Dependencies

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.19.0 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.3 (OK)
pygments >=2.0                 :  2.5.2 (OK)
pylint >=1.0                   :  1.9.4 (OK)
pyls >=0.34.0;<1.0.0           :  0.36.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.3 (OK)
qtconsole >=4.6.0              :  4.7.7 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
sphinx >=0.6.6                 :  1.8.5 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.2 (OK)

# Optional:
cython >=0.21                  :  0.29.14 (OK)
matplotlib >=2.0.0             :  3.3.2 (OK)
numpy >=1.7                    :  1.19.3 (OK)
pandas >=0.13.1                :  1.1.4 (OK)
scipy >=0.17.0                 :  1.5.4 (OK)
sympy >=0.7.3                  :  None (OK)
```

**Comment by eafpres** [comment]
Update---LSP logs (on gdrive, too large to paste):
https://drive.google.com/file/d/1v5DcAeFwHGSlinbnr_rpe3GZZQ5TXfJG/view?usp=sharing
https://drive.google.com/file/d/1u1_wmABCDU_q9SP4pkQn0-eoB9tJ-8z0/view?usp=sharing

**Comment by eafpres** [comment]
Update--this was fixed by installing Spyder 4.2.0 which accepted jedi 0.17.2

**Comment by ccordoba12** [comment]
Great!
